components/base-war/pom.xml

```diff
@@ -243,6 +244,7 @@ extension.repositories=maven-central:maven:https://nexus.phenotips.org/nexus/con
 extension.repositories=maven-phenotips:maven:https://nexus.phenotips.org/nexus/content/repositories/releases/
 extension.repositories=phenotips.org:xwiki:https://phenotips.org/rest/
 rendering.transformations=macro
+activeinstalls.pingURL=http://localhost:9200
```

Comment:
I'll change it once it passes first review, and I get access to phenotips.org to set up and secure elastic search.